requirements.txt

```diff
@@ -0,0 +1,19 @@
+accessify==0.3.1
```

Comment:
What are all these packages?
I see some that are not relevant for this task, please clean up

Reply:
You're right these are my global packages....
To be honest, first I haven’t fully understood the virtual environment. I couldn't activate it because I tried to do it in gitbash instead of terminal... 
In general, I agree that there is no logic in this file, I replaced it.